control-station/src/services/PodSocketClient.ts

```diff
@@ -72,40 +60,27 @@ class PodSocketClient {
 		this.socket.disconnect();
 	}
 
-	// Send a ping to the server
-	sendPing(): void {
-		this.socket.emit("ping", "ping", (ack: string) => {
-			console.log(`Server acknowledges ping with ${ack}`);
-		});
-	}
-
-	sendGreet(): void {
-		this.socket.emit("greet", "Hello from client", (ack: string) => {
-			console.log(`Server responds to greet with ${ack}`);
-		});
-	}
-
 	sendStop(): void {
-		this.socket.emit("stop", "Hello from client", (ack: string) => {
-			console.log(`Server responds to stop with ${ack}`);
+		this.socket.emit("stop", (response: string) => {
+			console.log("Server acknowledged:", response);
 		});
 	}
 
 	sendHalt(): void {
-		this.socket.emit("halt", "Hello from client", (ack: string) => {
-			console.log(`Server responds to stop with ${ack}`);
+		this.socket.emit("halt", (response: string) => {
+			console.log("Server acknowledged:", response);
 		});
 	}
 
 	sendLoad(): void {
-		this.socket.emit("load", "Hello from client", (ack: string) => {
-			console.log(`Server responds to stop with ${ack}`);
+		this.socket.emit("load", (response: string) => {
+			console.log("Server acknowledged:", response);
```

Comment:
Sorry to be nitpicky as well, but since you already ordered the fields in `ClientToServerEvents`, could you also do the same for the functions here and in `state_machine.rs` just for consistency's sake?

Reply:
yuh

Reply:
As well as in `StateMachine` please and thank you

Reply:
done